.github/workflows/cache-test-assets.yaml

```
@@ -35,7 +35,7 @@ jobs:
           echo "digest=$DIGEST" >> $GITHUB_OUTPUT
 
       - name: Restore and save test images cache
-        uses: actions/cache@v4
+        uses: actions/cache@1bd1e32a3bdc45362d1e726936510720a7c30a57 # v4.2.0
```

`actions/cache@v4` used version `v4.2.4` in the last workflow run:
https://github.com/aquasecurity/trivy/actions/runs/17289305448/job/49072685923#step:1:42
https://github.com/actions/cache/commits/v4.2.4/

```suggestion
        uses: actions/cache@0400d5f644dc74513175e3cd8d07132dd4860809 # v4.2.4
```

PS - I didn’t add comments for actions/cache in the other files.

---

can you update other `actions/cache` actions?